skytemple_files/dungeon_data/mappa_bin/item_list.py

```diff
@@ -41,7 +41,7 @@ class MappaItemCategory(Enum):
     THROWN_PIERCE = 0, 1, 6, [], [9]
     THROWN_ROCK = 1, 7, 4, [9], []
     BERRIES_SEEDS_VITAMINS = 2, 69, 40, [], [116, 117, 118]
-    FOODS_GUMMIES = 3, 109, 29, [116, 117, 118], []
+    FOODS_GUMMIES = 3, 109, 30, [116, 117, 118], []
```

Comment:
Could this cause issues when the patch isn't applied?

Reply:
Probably not, since item # 138 isn't in any other category. It may cause slight issues when adding it to a floor before applying the patch, but the impact should be minimal. Could maybe add check to the UI, but idk if it's worth it.